package com.arjoo.exceptions;

public record User(Integer id, String name) {}
